src/components/Feedback/Feedback.tsx

```diff
@@ -0,0 +1,215 @@
+  }
+}
+
+const getTextColor = (type: Type, palette: Palette): string => {
+  switch (type) {
+  case Type.Error:
+    return palette.error[600]
+  case Type.Special:
+    return '#7133CD'
+  case Type.Success:
+    return palette.success[600]
+  default:
+    return palette.text.neutral.main
+  }
+}
+
+/**
+ * UI component for displaying messages to the user.
+ * @returns {ReactElement} Feedback component
+ */
+export const Feedback = ({
+  alert: customAlert,
+  badge: customBadge,
+  icon: customIcon,
+  children: customChildren,
+  description,
+  title: customTitle,
+  type,
+}: FeedbackProps): ReactElement => {
+  const { palette } = useTheme()
+
+  const icon: ReactNode = useMemo(() => {
+    if (type === Type.Loading) {
+      return (
+        <div className={styles.spinner}>
+          <Spin indicator={<LoadingOutlined className={styles.loading} />} size="large" />
+        </div>
+      )
+    }
+
+    return (
+      <div data-testid="custom-icon">
+        <Icon color={getColor(type, palette)} component={customIcon ?? getIcon(type)} size={64} />
+      </div>
+    )
+  }, [customIcon, palette, type])
+
+  const title = useMemo(() => (
+    <div className={styles.titleWrapper}>
+      <div className={styles.title}>
+        <Typography.H2
+          color={getTextColor(type, palette)}
+          ellipsis={{ rows: 1, tooltip: { ...tooltipProps, title: customTitle } }}
+        >
+          {customTitle}
+        </Typography.H2>
+      </div>
+      {description && (
+        <div className={styles.description}>
+          <Typography.BodyS ellipsis={{ rows: 2, tooltip: { ...tooltipProps, title: description } }}>
+            {description}
+          </Typography.BodyS>
+        </div>
+      )}
+    </div>
+  ), [description, palette, customTitle, type])
+
+  const badge = useMemo(() => {
+    if (!customBadge) { return }
+
+    return (
+      <div className={styles.badge}>
+        <div className={styles.badgeIcon} data-testid="badge-icon">
+          <Icon color={palette.text.neutral.subtle} component={customBadge.icon} size={48} />
+        </div>
+        <div className={styles.badgeTitleWrapper}>
+          <div className={styles.badgeTitle}>
+            <Typography.H3 ellipsis={{ rows: 2, tooltip: { ...tooltipProps, title: customBadge.title } }}>
+              {customBadge.title}
+            </Typography.H3>
+            {customBadge.extra}
+          </div>
+          {customBadge.subtitle && (
+            <div className={styles.badgeSubtitle}>
+              <Typography.BodyS ellipsis={{ rows: 1, tooltip: { ...tooltipProps, title: customBadge.subtitle } }}>
+                {customBadge.subtitle}
+              </Typography.BodyS>
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }, [customBadge, palette.text.neutral.subtle])
+
+  const alert = useMemo(() => {
+    if (!customAlert) { return }
+
+    return (
+      <div className={styles.alert}>
+        <Alert isCompressed {...customAlert} />
```

Comment:
Are we sure we want to give the freedom to have uncompressed alerts?
Eventually i suggest to change the order of the props as follow `<Alert {...customAlert} isCompressed />`